CMakeLists.txt

```diff
@@ -665,12 +690,13 @@ if(NOT WIN32 OR ROCKSDB_INSTALL_ON_WINDOWS)
 
   install(DIRECTORY include/rocksdb COMPONENT devel DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}")
 
+  set(INTERFACE_INCLUDE_DIRECTORIES, "${CMAKE_INSTALL_INCLUDEDIR}")
+
   install(
     TARGETS ${ROCKSDB_STATIC_LIB}
     EXPORT RocksDBTargets
     COMPONENT devel
     ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
-    INCLUDES DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
```

Comment:
what remove this line?

Reply:
INCLUDES DESTINATION is supported only in cmake version > 2.8.12 and fails to compile on 2.8.11 and lower. The minimum cmake version is 2.6. This was overlooked.